packages/api/src/controllers/asset.ts

```diff
@@ -135,6 +138,25 @@ async function validateAssetPayload(
   };
 }
 
+function validateAssetPlaybackPolicy(playbackPolicy: Asset["playbackPolicy"]) {
+  if (playbackPolicy?.type == "lit_signing_condition") {
+    if (!playbackPolicy.unifiedAccessControlConditions) {
+      throw new UnprocessableEntityError(
+        `playbackPolicy.unifiedAccessControlConditions is required when using lit_signing_condition`
+      );
+    }
```

Comment:
This can be enforced on the schema through a `oneOf` definition of the playback policy. Then it's a required field only for this type of policy. Certainly not a required change tho, only throwing the idea.